src/utils/__tests__/alder.test.ts

```diff
@@ -115,6 +117,46 @@ describe('alder-utils', () => {
     })
   })
 
+  describe('getAlderPlus1Maaned', () => {
+    it('returnerer riktig alder med en måned mellom 0-11', () => {
+      const alder = getAlderPlus1Maaned({ aar: 63, maaneder: 3 })
+      expect(alder.aar).toBe(63)
+      expect(alder.maaneder).toBe(4)
+    })
+
+    it('returnerer riktig alder med måned 0', () => {
+      const alder = getAlderPlus1Maaned({ aar: 63, maaneder: 0 })
+      expect(alder.aar).toBe(63)
+      expect(alder.maaneder).toBe(1)
+    })
+
+    it('returnerer riktig alder med måned 11', () => {
+      const alder = getAlderPlus1Maaned({ aar: 63, maaneder: 11 })
+      expect(alder.aar).toBe(64)
+      expect(alder.maaneder).toBe(0)
+    })
+  })
+
+  describe('getAlderMinus1Maaned', () => {
+    it('returnerer riktig alder med en måned mellom 1-11', () => {
```

Comment:
Skal ikke `1-11` være `0-11`?
F.eks. 62 år 1 md minus 1 måned blir 62 år 0 md.

Reply:
Det var feil formulert av meg.
Jeg mente på ha en standard test for en hvilken som helst måned mellom 1-10
Så en test for de to ytterste alternativer: måned 0 og måned 11. (for å dekke logikken når det bikkes ett år)

Jeg korrigerer "description" for denne testen og den over.